httpx/concurrency/asyncio.py

```diff
@@ -14,15 +14,16 @@
 import typing
 from types import TracebackType
 
-from ..config import PoolLimits, TimeoutConfig
-from ..exceptions import ConnectTimeout, PoolTimeout, ReadTimeout, WriteTimeout
-from ..interfaces import (
+from .base import (
     BaseBackgroundManager,
     BasePoolSemaphore,
     BaseReader,
     BaseWriter,
     ConcurrencyBackend,
+    TimeoutFlag
```

Comment:
Apparently this is causing the Black linting check to fail:

```suggestion
    TimeoutFlag,
```